tools/cr/plaster.py

```diff
@@ -76,7 +76,12 @@ def save_if_changed(self, new_content: str, dry_run: bool = False) -> bool:
             return False  # No change detected
         logging.debug('Saving: %s', self.path)
         if not dry_run:
-            self.path.write_text(new_content, encoding='utf-8')
+            target_path = Path(self.path)
+            if (target_path.parts[0] == 'patches'):
+                target_path = Path.cwd() / Path(
+                    'patches/' + '-'.join(target_path.parts[1:]))
+            logging.debug('Writing to: %s', target_path)
+            target_path.write_text(new_content, encoding='utf-8')
```

Comment:
This doesn't seem to be correct. This is how this object is supposed to be created:

```py
        # Setup patch file (named based on source path, with dashes).
        self.patch = PathChecksumPair(
            PATCHES_PATH / f'{str(self.source).replace("/", "-")}.patch')
```

So we need to know what wrong about the line above in Windows

Reply:
Thanks, It seems self.source contains "\" instead of "/". We should be able to normalize it with as_posix().